src/buildercore/fastly/vcl/error-page.vcl.tpl

```diff
@@ -1,5 +1,5 @@
-if (obj.status == ${code}) {
-   set obj.http.Content-Type = "text/html; charset=us-ascii";
-      synthetic {"${synthetic_response}"};
-     return(deliver);
+if (${test}) {
+    set obj.http.Content-Type = "text/html; charset=us-ascii";
+    synthetic {"${synthetic_response}"};
```

Comment:
I wonder if we will need to escape this in case it contains a `"`?

Reply:
> Long strings are enclosed in {" ... "}. They may contain any character including ", newline and other control characters except for the NUL (0x00) character.

Reply:
:tada: